inst/tinytest/test_utilities.R

```diff
@@ -22,6 +24,8 @@ expect_error(quadfn(5, 1:5), "NULL value passed as symbol address")
 gc()
 expect_false(file.exists(environment(quadfn)$libLFile))
 
+if (isM1) exit_file("Skip remainer")
```

Comment:
Is there a function to skip a single test in tinytest? The CRAN check said that one test out of 30 failed, so we could keep the others.

Reply:
from
https://www.stats.ox.ac.uk/pub/bdr/M1mac/inline.out
```
  Running test_utilities.R..............   13 tests [0;31m1 fails[0m [0;34m2.8s[0m
  ----- FAILED[xcpt]: test_utilities.R<35--38>
   call| expect_error(current = moveDLL(quadfn, name = "testname", directory = tempdir(), 
   call| unload = TRUE, overwrite = TRUE), pattern = "file can not be copied both 'from' and 'to'")
   diff| No error
  Error: 1 out of 30 tests failed
  Execution halted
* checking PDF version of manual ... OK
```